tests/test_cases/test_cocotb/test_timing_triggers.py

```diff
@@ -42,8 +43,20 @@ async def test_function_reentrant_clock(dut):
         await timer
 
 
-@cocotb.test()
+# Xcelium/VHDL does not correctly report the simulator precision.
+# See also https://github.com/cocotb/cocotb/issues/3419
+# NVC and GHDL do not support setting precision and always use 1 fs.
```

Comment:
@nickg Is it an intentional design choice that there seems to be no way to set the time precision, or is it worth filing a feature request for it?

@tgingold The same question for you on GHDL, only that I found an issue a while back that stated that GHDL internally always uses 1 fs and is designed around that assumption. Still checking with you to be sure.

Reply:
No, GHDL can use precision other than 1 fs.

Reply:
@tgingold Good to know! How does one configure that from the command line?

Reply:
There is the `--time-resolution` option, but in general it's automatic (use the least precision)

Reply:
@tgingold Excellent, thanks! I'll follow up with a PR to use this switch. (We have a test where we need to force a specific resolution, hence the need for a switch.)

Reply:
There's an open issue at nickg/nvc#607 but I've not got round to implementing it yet.

Reply:
@tgingold FYI: https://github.com/cocotb/cocotb/pull/3622 has the update for GHDL.

Reply:
@nickg Thanks! I added a link to the open issue in https://github.com/cocotb/cocotb/pull/3622.